deploy/resources/v0.11.1/addons/05-consolecli/console_cli_download_tkn.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: console.openshift.io/v1
+kind: ConsoleCLIDownload
+metadata:
+  name: tkn
+spec:
+  description: The OpenShift Pipeline client `tkn` is a CLI tool that allows you to manage OpenShift Pipeline resources.
+  displayName: tkn - OpenShift Pipeline Command Line Interface (CLI)
+  links:
+    - href: https://mirror.openshift.com/pub/openshift-v4/clients/pipeline/0.9.0/tkn-linux-amd64-0.9.0.tar.gz
+      text: Download tkn for Linux
+    - href: https://mirror.openshift.com/pub/openshift-v4/clients/pipeline/0.9.0/tkn-macos-amd64-0.9.0.tar.gz
+      text: Download tkn for macOS
+    - href: https://mirror.openshift.com/pub/openshift-v4/clients/pipeline/0.9.0/tkn-windows-amd64-0.9.0.zip
+      text: Download tkn for Windows
```

Comment:
I think the URL should have `tkn` instead of `pipeline`.

see: https://coreos.slack.com/archives/CG5GV6CJD/p1585923659334900
cc: @siamaksade

Reply:
@navidshaikh I am not sure why ? `kn` is under `serverless` isn't it ?

Reply:
@vdemeester : I wrote `tkn`  :-)

Reply:
Yes i also thought the same and used `pipeline`

Reply:
There was a discussion whether to use `tkn` or `pipeline` on the slack thread I referenced.

Reply:
@navidshaikh I am talking about `kn` : it follow the same pattern https://mirror.openshift.com/pub/openshift-v4/clients/serverless/0.13.1/

for `kn` : folder is `serverless`, binary/tarball is `kn`
for `tkn` : folder is `pipeline`, binary/tarball is `tkn`

Reply:
Regarding `serverless`, IMO we should've used `kn` as well there, but now its out of the door..

since all the other binaries at https://mirror.openshift.com/pub/openshift-v4/clients/ are using their `CLI` name as the dir name. Similarly we can begin hosting it with `tkn`.

Reply:
I'll leave the actual naming discussion to the team though. :-)

Reply:
Alright, that make sense to me too :stuck_out_tongue: